src/components/layout/AvailabilityHeader/AvailabilityHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import './styles.css';
+function AvailabilityHeader({
```

Comment:
i think we could abstact this even more, and swap `availability` for `isSuccessful` or `isValid`
```suggestion
function SearchBarHeader({
```